src/main/java/com/pablissimo/sonar/TsLintSensor.java

```diff
@@ -96,9 +96,28 @@ else if (config.getConfigFile() == null && config.getPathToTsConfig() == null) {
 
             String lowerFilePath = filePath.toLowerCase();
 
+
+
             if (!fileMap.containsKey(lowerFilePath)) {
                 LOG.warn("TsLint reported issues against a file that wasn't sent to it - will be ignored: " + lowerFilePath);
-                continue;
+
```

Comment:
K, what might be easiest then is a fallback attempt at a case-insensitive check but only on Windows (though that seems hacky). I don't know why I can't get it to give me a lower-case drive spec though; can you do a `pwd` from the folder and see whether it upper-cases the drive letter or not?